assets/js/blocks/order-confirmation/status/block.json

```diff
@@ -7,7 +7,38 @@
 	"keywords": [ "WooCommerce" ],
 	"supports": {
 		"multiple": false,
-		"align": [ "wide", "full" ]
+		"align": [ "wide", "full" ],
+		"html": false,
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontFamily": true,
+			"__experimentalTextDecoration": true,
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalTextTransform": true,
+			"__experimentalWritingMode": true,
```

Comment:
Similar to `__experimentalFontWeight`, I'm unable to see `__experimentalWritingMode` on the site.

Reply:
https://github.com/woocommerce/woocommerce-blocks/pull/10164/commits/246b75abdc1bdd7fa348e64867d41fe489f4538d